frontend/components/MainPage/sections/RewardsSection/index.tsx

```diff
@@ -28,18 +28,24 @@ const DisplayRewards = () => {
   const {
     availableRewardsForEpochEth,
     isEligibleForRewards,
-    isStakingRewardsDetailsFetched,
+    isStakingRewardsDetailsLoading,
+    isStakingRewardsDetailsError,
   } = useReward();
   const { isLoaded: isBalancesLoaded } = useBalanceContext();
   const reward = getFormattedReward(availableRewardsForEpochEth);
 
   const earnedTag = useMemo(() => {
-    if (!isStakingRewardsDetailsFetched) return <Skeleton.Input size="small" />;
+    if (isStakingRewardsDetailsLoading && !isStakingRewardsDetailsError)
```

Comment:
just to ensure the tag is not flickering, I need to double check with Iason what might be the error there